Private/Format-Result.ps1

```diff
@@ -55,33 +63,41 @@ function Format-Result {
 
 
         if ($Mode -eq 0) {
+            # TODO Refactor this
             switch ($UniqueIssue) {
-                'DETECT'      { $Issue | Format-Table Technique, Name, Issue -Wrap }
-                'ESC1'        { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC1'        { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC1'        { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC4'        { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC5'        { $Issue | Format-Table Technique, Name, Issue -Wrap }
-                'ESC6'        { $Issue | Format-Table Technique, Name, Issue -Wrap }
-                'ESC8'        { $Issue | Format-Table Technique, Name, Issue -Wrap }
-                'ESC11'       { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC13'       { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-                'ESC15/EKUwu' { $Issue | Format-Table Technique, Name, Enabled, Issue -Wrap }
-
+                { $_ -in @('DETECT', 'ESC6', 'ESC8', 'ESC11') } {
+                    $Issue |
+                    Format-Table Technique, @{l = 'CA Name'; e = { $_.Name } }, @{l = 'Risk'; e = { $_.RiskName } }, Issue -Wrap |
+                    Write-HostColorized -PatternColorMap $RiskTable -CaseSensitive
+                }
+                { $_ -in @('ESC1', 'ESC2', 'ESC3', 'ESC4', 'ESC13', 'ESC15/EKUwu') } {
```

Comment:
I just found myself thinking that for the sake of consistency, we should refer to all of these by their ESC*nn* names and not their fun code names. Again, purely for consistency, unless/until we find that people recognize some of these *more* easily by their names than their ESC*nn* reference.

Reply:
You're probably right. LS2!